### ISSUE TITLE
Zmizení kardinalit u zdroje

### ISSUE THREAD

**Issue by IR158** [opendata-mvcr/sgov-assembly-line#113]
Dobrý den,
Ontographer - pracovní prostor Veřejné zakázky.
Poradili byste mi prosím, čím může být způsobeno, že u většiny prvků zmizely kardinality u zdroje a u jiných zase vazby (Zadávací postup na záložce Výsledek zadávacího postupu)? U prvku Druh části veřejné zakázky na záložce Základní atributy zadávacího postupu zase zmizel Typový stereotyp. Se schématy jsme v posledních měsících nepracovali.

**Comment by IR158** [comment]
Zadávací postup byl navázán na Výsledek zadávacího postupu. S kardinalitami je to zvláštní, posílám náhled, jak to vidím já. Když pak přepnu do kompaktního modu, tak jsou téměř všude 0.
![kardinality](https://user-images.githubusercontent.com/97511616/170281741-fbe82410-2ba7-4841-bc80-02871ae74401.png)

**Comment by MichalMed** [comment]
Tohle není kompaktní mód, ale full mód. V kompaktním módu to vypadá takhle:
![Subjekty (1)](https://user-images.githubusercontent.com/3779672/170428516-869c2615-698f-49a0-8255-517f4220e56e.png)
Ale kardinality by stejně zmizet neměly. Stejně tak vazby. Předám to dál.

**Comment by IR158** [comment]
Ano, omlouvám se, asi jsem se nepřesně vyjádřila, obrázek, který jsem posílala, měl znázornit, že v úplném pohledu kardinality na mnoha místech chybí, což pak způsobí, že po přepnutí do kompaktního modu se objevují 0. Kardinality znovu doplním a snad to bude v pořádku. Děkuji

**Comment by MichalMed** [comment]
@IR158 mělo by být opraveno a nasazeno, prosím o otestování

**Comment by MichalMed** [comment]
některé katdinality se tam zase objevily, ale všude stejně nejsou
